plugins/source/pagerduty/Makefile

```diff
@@ -0,0 +1,35 @@
+# Test unit
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+# Install tools
+.PHONY: install-tools
+install-tools:
```

Comment:
either depend on this target in gen-mocks and don't install mockgen explicitly there, or remove this target

Reply:
not a makefile-guru - I just copy-pasted this from `aws`.
fixing this everywhere seems out-of-scope of the pagerduty PR.